WMIAdventure/backend/WMIAdventure_backend/IngameUsers/businesslogic/experience/CachedLevelBoundaries.py

```diff
@@ -0,0 +1,23 @@
+from IngameUsers.businesslogic.experience.formulas import level_to_exp
+
+
+class CachedLevelBoundaries:
+    """
+    Caching class that tells you what experience is required for a specific level.
+    This is cached at runtime using values calculated by exp formulas.
+    """
+    boundaries_cache = {}
+    
+    # injectable
+    level_to_exp_func = level_to_exp
+
+    @staticmethod
+    def get_min_exp_for_level(level):
```

Comment:
nazwa funkcji troche myląca, bo to zwraca ile trzeba mieć expa na następny level, tak? może coś w stylu `get_min_exp_for_next_level(current_level)`?

Reply:
Nie, ta funkcja robi dokładnie to co opisuje, zwraca ile expa jest potrzebne żeby mieć dany level

Reply:
to czemu na pierwszym levelu dostaje 10?

Reply:
mogłem coś źle wpisać, spróbuje jeszcze raz

Reply:
hm dalej tak mam
![image](https://user-images.githubusercontent.com/43492764/146252725-f849d0ed-795b-4c22-b01c-b93bbf090afd.png)